setup.py

```diff
@@ -786,20 +453,44 @@ def load_config(config_path: Path) -> Dict[str, str]:
     return parse_config_vars(config_data)
 
 
-def _build_spec(preset_name: str, fork: str,
-                source_files: Sequence[Path], preset_files: Sequence[Path], config_file: Path) -> str:
+def build_spec(preset_name: str, fork: str, source_files: Sequence[Path], preset_files: Sequence[Path],
+               config_file: Path, spec_dir: Path, spec_yaml: Dict) -> str:
     preset = load_preset(preset_files)
     config = load_config(config_file)
     all_specs = [get_spec(spec, preset, config) for spec in source_files]
 
-    spec_object = all_specs[0]
-    for value in all_specs[1:]:
+    python_prefix = []
```

Comment:
In general, I think another pass needs to be taken at handling including and combining the `python_prefix`es and suffixes. The was it is currently done does not match the pattern of the rest of this pyspec_builder, that of extracting the prefix into a `spec_object` and then having a reducer function to combine spec objects back into a single object.

IMO, the prefixes & suffixes for each phase should be extracted in `build_spec()` instead of being added separately in the `spec.yaml`. If this pattern is not used here, then I think we should remove the combiner functionality that is not used and this anti-patter should be explained as a comment.